### DISCUSSION TITLE
[Feature Request] Panasonic/MKE Interface

### DISCUSSION THREAD

**Discussion by mw308** [86Box/86Box#3054] [Ideas]
Hello

Would it be possible to implement the Panasonic/MKE CD-ROM interface that was present on SoundBlaster/compatibles?  More info if required is available here:

`http://www.chiark.greenend.org.uk/~theom/electronics/panasoniccd.html`

Thanks!

**Comment by waltje** [comment]
Haha, I just started working on this a few days ago! :)

--waltje

--------------------------------------------
On Mon, 10/15/18, mw308 <notifications@github.com> wrote:

 Subject: [86Box/86Box] [Feature Request] (#327)
 To: "86Box/86Box" <86Box@noreply.github.com>
 Cc: "Subscribed" <subscribed@noreply.github.com>
 Date: Monday, October 15, 2018, 10:30 AM
 
 Hello
 Would it be possible to implement the Panasonic/MKE
 CD-ROM interface that was present on
 SoundBlaster/compatibles?  More info if required is
 available here:
 http://www.chiark.greenend.org.uk/~theom/electronics/panasoniccd.html
 Thanks!
 
 —
 You are receiving this because you are subscribed to this
 thread.
 Reply to this email directly, view it on
 GitHub, or mute
 the thread.

**Comment by mw308** [comment]
Ah great-can’t wait to try it!

**Comment by mw308** [comment]
Hi Waltje

Do you know when this is likely to be commited?

Thanks

**Reply by OBattler** [reply]
This is now finally done. It's available in the current Jenkins builds.

**Reply by mw308** [reply]
Yes I've been using it on the last few commits and I'm extremely grateful, thank you :)

The sound card I had this on was a MediaVision ProSonic 16 (ISA-16) using the Jazz 16 chip (SB-compatible), is there any chance of having that one in 86box at any point please?  Would you like me to make another discussion thread for it?

Thanks again!

**Reply by OBattler** [reply]
Yes, please do make another discussion thread for it.
